x-pack/plugin/deprecation/qa/early-deprecation-rest/build.gradle

```diff
@@ -0,0 +1,41 @@
+import org.elasticsearch.gradle.util.GradleUtils
```

Comment:
this is basically a copy of x-pack/plugin/deprecation/qa/rest/build.gradle
can I move shared bits to common module?

Reply:
I had to create that module, so that I could make sure that `EarlyDeprecationIndexingIT` is the test that runs first thing after test cluster starts